pymc3/plots/kdeplot.py

```diff
@@ -3,15 +3,34 @@
 from scipy.signal import gaussian, convolve
 
 
-def kdeplot(trace_values, label=None, alpha=0.35, shade=False, ax=None,
-            **kwargs):
+def kdeplot(values, label=None, shade=0, ax=None, kwargs_shade={}, **kwargs):
+    """
```

Comment:
nice ;-)

Reply:
Shouldn't use mutable default kwargs.

Reply:
Fixing the dangerous defaults in another PR, along with activating the pylint check...

Reply:
Sorry about that, @twiecki thanks for pointing that out and @ColCarroll thanks for fixing it!